lib/wordpress.js

```diff
@@ -0,0 +1,48 @@
+var _ = require('underscore');
+var params = require('querystring').parse(window.location.search.split('?').pop());
+
+module.exports = {
```

Comment:
`module.exports` is already an object, for readability and consistency with the rest of the package, I suggest `module.exports.getPostPreviewView = function (view) {`.